app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,26 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - i = 0;
+  - @conference.booths.confirmed.each_slice(3) do |slice|
+    .row.row-centered
+      - slice.each do |booth|
+        .col-md-4.col-sm-4.col-centered.col-top
+          .thumbnail
+            - if booth.logo_link
+              = link_to booth.website_url, class: 'thumbnail' do
+                = image_tag booth.picture.large.url
+            .caption
+              %h3.text-center
+                = booth.title
+              %p.text-center.text-muted
+                = link_to "#show_descrition_#{i}", "data-toggle"=>"collapse" do
+                  learn more
+              .collapse{ id: "show_descrition_#{i}" }
+                = markdown(booth.description)
+        - i = i + 1
```

Comment:
I am aware but as I explained, since there is the `|slice|`, `each.with_index` doesn't work. An other option could be  a for loop where the `- i = 0` currently is and the rest of the code inside in this loop.